### PR TITLE
Removes Ladders

### LINKED ISSUE #13391: Alt-Click tabs on tiles aren't closed when you climb Jacob's Ladder
**Issue Description**:
Climbing or descending Jacob's Ladder doesn't close Alt-Click tabs, allowing you to interact with objects or mobs at the bottom of the ladder while you're at the top of the ladder.

**What did you expect to happen**: 
The Alt-Click window to move with my character to the different z-level or close.

**What happened instead**:
It doesn't close or move, allowing you to interact with anything on the tile except for the turf.

**Why is this bad/What are the consequences**:
You can teleport from the top of the ladder to the bottom by Alt-Clicking the tile the ladder is on, or attack or interact with things at the bottom while at the top.

**Steps to reproduce the problem**:
Obtain or spawn in the Jacob's Ladder item.
Use it.
Ensure you have Spess protection.
Alt-Click a tile with a mob or item on it before climbing to the top of the ladder.
Interact with that mob or item.

**When did the problem start happening**:
N/A

**Extra information**:
Haven't tried this with regular ladders. Does Paracode have regular ladders?

### PR DESCRIPTION
**Pull request by DGamerL**
## What Does This PR Do
Fixes #13391
Removes Jacob's ladder.
Removes ladders.

## Why It's Good For The Game
It is a very very silly item, something that was realized in 2020 already. It has been added in 2019, and was removed from tendril loot in 2021. It was only used recently for ladders such as the anchor in the beach gateway ruin.

## Testing
Booted up a server, couldn't spawn it in.

## Changelog
:cl:
del: You can't get Jacob's Ladder out of the nannybag anymore.
/:cl:

### CODE REVIEW

**Comment by spacegeese** [comment]
what did ladders do anyways?

**Comment by procdrone** [comment]
> what did ladders do anyways?

it allowed you to move between z levels by climbing up and down.

**Comment by Henri215** [comment]
The plan to add a ladder for decoration purpose only is still up?

**Comment by DGamerL** [comment]
> The plan to add a ladder for decoration purpose only is still up?

Yep, forgot about this PR, gimme a bit

**Review by Contrabang — APPROVED** [review]
lgtm

**Review by DGamerL — COMMENTED** [review]
TODO: fix the fucking althland ruin

**Review by Contrabang — CHANGES_REQUESTED** [review]
oops, my bad

**Review by Sirryan2002 — APPROVED** [review]

File: _maps/map_files/RandomRuins/LavaRuins/lavaland_surface_althland_excavation.dmm [hunk 1]
```diff
@@ -1145,12 +1145,9 @@
 /area/lavaland/surface/outdoors)
 "Es" = (
 /obj/structure/lattice/catwalk/mining,
-/obj/structure/ladder/unbreakable/dive_point{
-	desc = "The shattered remnants of a ladder hanging over the pitch-black chasm, providing no safe route to the abyss beneath. Its fractured rungs suggest an inevitable plunge, as if the remainder might soon join the void.";
-	name = "shattered ladder"
-	},
```
Comment (Contrabang) [inline _maps/map_files/RandomRuins/LavaRuins/lavaland_surface_althland_excavation.dmm:1151]: I suggest you keep this desc and name for the ladder, and change the name of the path too

**Review by Contrabang — CHANGES_REQUESTED** [review]

**Review by Burzah — APPROVED** [review]

**Review by S34NW — APPROVED** [review]

**Pull request merged**